program/rust/src/rust_oracle.rs

```diff
@@ -322,6 +323,32 @@ pub fn upd_product(
     Ok(SUCCESS)
 }
 
+pub fn set_min_pub(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    instruction_data: &[u8],
+) -> OracleResult {
+    let cmd = load::<cmd_set_min_pub_t>(instruction_data)?;
+
+    pyth_assert(
+        instruction_data.len() == size_of::<cmd_set_min_pub_t>(),
```

Comment:
i'm pretty sure this is unnecessary (should be covered by load), but i'm being defensive in literally copying the C.